terraform/deployments/variables/integration/common.tfvars

```diff
@@ -22,6 +22,12 @@ eks_private_subnets = {
   c = { az = "eu-west-1c", cidr = "10.1.32.0/22" }
 }
 
+rabbitmq_subnets = {
```

Comment:
Do we actually want dedicated subnets for RabbitMQ? Is there some advantage to doing that that outweighs the admin hassle? Bear in mind at some point we'll have RDS databases and such as well.

Reply:
@sengi thanks for your thoughts, I thought this would be consistent with how we have configured other managed services (elasticache, rds). I saw we have [private_subnet_cidrs](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-networking/production/govuk.tfvars#L19) of size `/24`, should we reuse those? Maybe you could talk me through the hassle we've had with this in the past, would love to learn 😃